config/maintainers.json

```diff
@@ -130,6 +130,16 @@
       "link_url": null,
       "avatar_url": null,
       "bio": null
+    },
+    {
+      "github_username": "andres-zartab",
+      "alumnus": true,
```

Comment:
This would mean you were a maintainer and have stopped being one.

```suggestion
      "alumnus": false,
```

Reply:
Oh, I didn't got that right!

Reply:
No problem. I did the same 😁